app/src/main/cpp/skyline/gpu/trait_manager.cpp

```diff
@@ -79,6 +79,12 @@ namespace skyline::gpu {
             enabledFeatures2.unlink<vk::PhysicalDeviceVertexAttributeDivisorFeaturesEXT>();
         }
 
+        if (supportsLastProvokingVertex) {
```

Comment:
Can drop the {} here, see our style guide

Reply:
Done.